megamek/src/megamek/common/Player.java

```diff
@@ -417,6 +418,7 @@ public int getBV() {
      */
     @Override
     public int getFledBV() {
+        //TODO: I'm not sure how squadrons are treated here - see getBV()
```

Comment:
Note that I changed the getBV() function you updated to a stream() a few days ago because it counted Squadrons twice. With the FledBV I just wanted to note that the same problem applied here maybe but Im not about to figure that out now.